### ISSUE TITLE
More notification options

### ISSUE THREAD

**Issue by thiagomdiniz** [glowroot/glowroot#389]
At the moment the only notification channel available is via email, right?

It would be very interesting to add a channel that would allow communication with Zabbix, for example. On the internet there are several examples of implementing Zabbix Sender in Java ([here](https://www.zabbix.org/wiki/Docs/protocols/zabbix_sender/1.8/java_example) or [here](https://github.com/hengyunabc/zabbix-sender)) and it would be great if Glowroot allowed this integration.

Thanks

**Comment by trask** [comment]
Hi @thiagomdiniz, thanks for posting! Are you looking to send alerts to Zabbix or to send metrics to Zabbix?

**Comment by thiagomdiniz** [comment]
Hi @trask !

I believe that sending metrics would be ideal, because with this the alerts can be configured in Zabbix. But in parallel we have the Glowroot monitoring capabilities that work at the application level, so I believe it would initially be best to create a new notification channel that acts as a [zabbix_sender](https://www.zabbix.com/documentation/3.4/manpages/zabbix_sender), letting you configure the Zabbix server address, the item name, host and value to be sent.

In this [link](https://github.com/marcelogwolff/wso2-zabbix_sender-connector/tree/master/org.wso2.carbon.connector.zabbix-sender/src/main/java/org/wso2/carbon/connector/zabbix_sender) there is an implementation of zabbix_sender in Java that we use in a project with WSO2.

Thanks!

**Comment by trask** [comment]
Hi @thiagomdiniz, sorry for the late reply. Are you using the central collector or the embedded collector? 
Would you want to send 1-min (rolled up) gauge values to Zabbix, or 5-second raw gauge values?

**Comment by thiagomdiniz** [comment]
@trask I am using the central collector.
I think sending between 1 and 5 minutes would be great. But I do not know if I was clear. The initial idea would be to use the glowroot's own monitoring, allowing to send a data to zabbix (trapper) instead of sending an email.

**Comment by trask** [comment]
Hi @thiagomdiniz, sorry (again) for the late reply, are you looking for something like #469 (but for Zabbix)?

**Comment by trask** [comment]
Hi @thiagomdiniz, I have Zabbix running locally, and mostly understand this now.  For the `value`, should Glowroot send `1` when the alert opens, and `0` when the alert closes?

**Comment by gamefundas** [comment]
@trask  webhook support will make a lot of sense as people can wire any alerting framework/implementations to it.

**Comment by ReillyProcentive** [comment]
Yeah, webhook integration would be ideal so that people could wire this up to something like Pagerduty.

**Comment by Never-lab** [comment]
Wishlist: **more notification channels** beyond what's built-in (email/etc.). Partial overlap with #586 / webhook-style asks. Enhancement, not a defect.

@nowheresly — please label `enhancement`.

**Comment by Never-lab** [comment]
Nice idea — thanks for filing it.

Closing as **obsolete**: open for years with no concrete design/PR path against current 0.14.x, and we are not planning to implement it as originally scoped. If this is still needed, please open a fresh issue with a short proposal against a current Glowroot build.

**Brief analysis:** Extra notification channels (e.g. Zabbix) are still desirable in theory, but this tracker is too broad/old; revisit only with a concrete channel design.
